tests/unit/schema/wrappers/test_message.py

```diff
@@ -124,6 +125,18 @@ def test_get_field_nonterminal_repeated_error():
         assert outer.get_field('inner', 'one') == inner_fields[1]
 
 
+def test_resource_path():
+    options = descriptor_pb2.MessageOptions()
+    patterns = options.Extensions[resource_pb2.resource].pattern
+    patterns.append("kingdom/{kingdom}/phylum/{phylum}/class/{klass}")
```

Comment:
nit: let's change the static parts to plurals to be consistent with (most) real APIs.

Reply:
Done.